Src/coffeescript/cql-execution/src/elm/query.coffee

```diff
@@ -28,6 +27,8 @@ module.exports.Without = class Without extends With
   exec: (ctx) ->
     !super(ctx)
 
+module.exports.Sort = class Sort extends UnimplementedExpression
+
```

Comment:
I'd like to leave it in so even if we don't implement it, we know what aspects of ELM are not surfaced in CQL and not supported in the engine.  Perhaps we should add comments to things like this?  Do you want me to do that before you merge?